pilot/cmd/pilot-agent/status/util/stats.go

```diff
@@ -24,21 +24,27 @@ import (
 )
 
 const (
-	statCdsUpdates = "cluster_manager.cds.update_success"
-	statLdsUpdates = "listener_manager.lds.update_success"
+	statCdsUpdatesSuccess   = "cluster_manager.cds.update_success"
+	statCdsUpdatesRejection = "cluster_manager.cds.update_rejected"
+	statLdsUpdatesSuccess   = "listener_manager.lds.update_success"
+	statLdsUpdatesRejection = "listener_manager.lds.update_rejected"
 )
 
 // Stats contains values of interest from a poll of Envoy stats.
 type Stats struct {
-	CDSUpdates uint64
-	LDSUpdates uint64
+	CDSUpdatesSuccess   uint64
+	CDSUpdatesRejection uint64
+	LDSUpdatesSuccess   uint64
+	LDSUpdatesRejection uint64
 }
 
 // String representation of the Stats.
 func (s *Stats) String() string {
-	return fmt.Sprintf("cds updates: %d, lds updates: %d",
-		s.CDSUpdates,
-		s.LDSUpdates)
+	return fmt.Sprintf("cds updates: %d success, %d rejected; lds updates: %d success, %d rejected",
```

Comment:
Nit: `success` and `rejected` are in different tense. Perhaps use `successful` or `accepted` instead of `success`?

Reply:
Adapted

Reply:
As long as we are taking `rejected`into account, assigning to `0` is important, I think. Since a missing `rejected`state should not be considered a problem, right?